service/controller/clusterapi/v29/resource/tccp/testdata/case-0-basic-test.golden

```diff
@@ -194,6 +194,39 @@ Resources:
       GatewayId:
         Ref: InternetGateway
   
+  ApiInternalLoadBalancer:
+    Type: AWS::ElasticLoadBalancing::LoadBalancer
+    DependsOn:
+      - VPCGatewayAttachment
+    Properties:
+      ConnectionSettings:
+        IdleTimeout: 1200
+      HealthCheck:
+        HealthyThreshold: 2
+        Interval: 5
+        Target: TCP:443
+        Timeout: 3
+        UnhealthyThreshold: 2
+      Instances:
+      - !Ref rsc-ac0dc01
+      Listeners:
+      
```

Comment:
Can we please get rid of the empty lines here? We put quite some effort into making the templates a bit more useful, readable and clean. The templates of existing clusters are super messy and I would like to make this a little bit better even when go templates are not really helpful with this.

Reply:
I'm not sure what to answer here. This file had empty lines before my commit. I don't know what are the rules here, so just followed existing file structure, e.g. https://github.com/giantswarm/aws-operator/blob/master/service/controller/clusterapi/v29/resource/tccp/testdata/case-0-basic-test.golden#L229-L230

Reply:
The "rule" is to try to make it better.

Reply:
Sure
I’m fine to make it better
As I said I just don’t know in current case. What is better. If dropping all empty lines is better - I’ll gladly do that.

Reply:
Ok, I've checked this and this file is generated from test.